.changeset/rare-falcons-travel.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/react': major
```

Comment:
This is a breaking change as the prop is required; please do let me know if a different release strategy would be better.

Reply:
Only the `title` prop being required for now